src/example/java/com/box/sdk/example/AccessAsAppUser.java

```diff
@@ -24,8 +24,8 @@ private AccessAsAppUser() {
     }
 
     public static void main(String[] args) throws IOException {
-        // Turn off logging to prevent polluting the output.
-        Logger.getLogger("com.box.sdk").setLevel(Level.OFF);
+        // Limit logging WARNINGS to prevent polluting the output.
```

Comment:
This is a little bit confusing. You are not limiting warnings by setting logging level Level.WARNING. You are limiting messages with lower level than warning. So I would rephrase to:
`Limit logging messages to prevent polluting the output.`